libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -612,6 +616,22 @@ export class RushConfiguration {
     this.subspacesConfiguration = SubspacesConfiguration.tryLoadFromDefaultLocation(this);
     this.subspacesFeatureEnabled = !!this.subspacesConfiguration?.subspacesEnabled;
 
+    if (this.subspacesFeatureEnabled && rushConfigurationJson.ensureConsistentVersions !== undefined) {
+      throw new Error(
+        'When using subspaces, the ensureConsistentVersions config is now defined in the common-versions.json file, ' +
```

Comment:
Is the common-versions filename not also in a constant?

Reply:
Fixed